pkgs/by-name/sw/sword/package.nix

```diff
@@ -5,9 +5,16 @@
   pkg-config,
   icu,
   clucene_core,
+
+  bzip2,
+  cmake,
   curl,
+  xz,
 }:
 
+let
+  isWindows = stdenv.hostPlatform.isWindows;
```

Comment:
This actually goes better with `inherit`, but IMO I wouldn't use the alias, I would use it directly.

Reply:
Definitely should have used `inherit`. It gets used so many times through the file that I just found it much tidier to shorten the form of it. If there's a strong bias against it, I don't mind restoring it. It just saved my fingers in the typing process.

Reply:
It makes few to no sense to use the `isWindows` shortening alias but not using `isUnix`.

Reply:
This was reverted to its long form throughout.